code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -347,7 +347,7 @@
 		if(QDELETED(L) || L.stat == DEAD)
 			return
 
-		to_chat(L, "<span class='italics'>You hear a voice in your head saying: </span><span class='abductor'>[message]</span>")
+		to_chat(L, "<span class='italics'>You hear a voice in your head saying: </span><span class='abductor'>[message]</span></span>")
```

Comment:
There's two `<span`s and three `</span`s, is that intentional?

Reply:
Nope MB